Constitution.txt

```diff
@@ -1,490 +1,264 @@
+2: No Senator or Representative shall, during the Time for which he was elected, be appointed to any civil Office under the Authority of these United States, which shall have been created, or the Emoluments whereof shall have been increased during such time; and no Person holding any Office under these United States, shall be a Member of either House during his Continuance in Office.
 
 Section 7
-1: All Bills for raising Revenue shall originate in the House of Representatives; but the Senate may propose or concur with Amendments as on other Bills.
+1: All Bills which require the spending of monies shall in their text provide for raising of Revenues as may allow for the spending, and shall originate in the House of Representatives. Such bills may, with the assent of the House, be amended by the Senate, but no amendment shall be made as would change the central character of the Bill from its original introduction.
 
-2: Every Bill which shall have passed the House of Representatives and the Senate, shall, before it become a Law, be presented to the President of the United States; If he approve he shall sign it, but if not he shall return it, with his Objections to that House in which it shall have originated, who shall enter the Objections at large on their Journal, and proceed to reconsider it. If after such Reconsideration two thirds of that House shall agree to pass the Bill, it shall be sent, together with the Objections, to the other House, by which it shall likewise be reconsidered, and if approved by two thirds of that House, it shall become a Law. But in all such Cases the Votes of both Houses shall be determined by yeas and Nays, and the Names of the Persons voting for and against the Bill shall be entered on the Journal of each House respectively. If any Bill shall not be returned by the President within ten Days (Sundays excepted) after it shall have been presented to him, the Same shall be a Law, in like Manner as if he had signed it, unless the Congress by their Adjournment prevent its Return, in which Case it shall not be a Law.
+2: Every Bill which shall provide for the funding of actions of the Federal Government of these United States prescribed by this Constitution which has been presented first to the House of Representatives, received the approval and signature of two-thirds of their number, presented to the Senate, and received the approval and signature of two-thirds of their number, shall be presented to the President for his approval. If he should approve the Bill by signature, it shall become Law; if not, or if he shall not provide his approval within 14 calendar days of the Senate's approval, it shall be returned to the House of Representatives, and if three-fourths of the Representatives approve, it shall become law. Bills passed into law shall retain force of law for a period to be defined in the Bill, but not to exceed six years.
```

Comment:
For bills that lie within the scope of the powers of Congress, I raise the requirement from a simple majority to a 2/3 majority. This should help to avoid the national partisanship which has gripped America since the founding, where the country organizes into two camps of similar size, and then trades back and forth over simple control of the Congress. That, along with the expanded House and Senate, the additional restrictions on Federal power, and the Senate elections returned to the State Legislatures, should help to move the two-party dynamic down to the state level, like we currently see in the European Union. MEP's of different national parties form coalitions within the European Parliament, but their loyalties are tied much more to their national interest than to European interests.

The other important change here is a hard time limit on ALL Bills of 6 years. This serves several purposes:

- It allows for the entire Legislature to be replaced before the Bill is reconsidered. If a Bill is particularly egregious, at least one third of the Congress will be replaced, and a refresh to the Bill's time limit will fail.
- Along with the restriction that Congress may not sit for more than 100 days, it forcibly restricts the ability of the Congress to legislate. This will force Representatives to prioritize, and only bring into national policy the most essential elements of legislation. All other issues will be left to the States not merely because this document says so, but because the Congress won't have time to deal with them.